pkg/scheduler/logging.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scheduler
+
+import (
+	"github.com/go-logr/logr"
+	"k8s.io/klog/v2"
+
+	"sigs.k8s.io/kueue/pkg/cache"
+)
+
+func logAdmissionAttemptIfVerbose(log logr.Logger, e *entry) {
+	logV := log.V(3)
+	if !logV.Enabled() {
+		return
+	}
+	args := []any{
+		"workload", klog.KObj(e.Obj),
+		"clusterQueue", klog.KRef("", e.ClusterQueue),
+		"status", e.status,
+		"reason", e.inadmissibleMsg,
+	}
+	if log.V(5).Enabled() {
+		args = append(args, "nominatedAssignment", e.assignment)
+	}
+	logV.Info("Workload evaluated for admission", args...)
+}
+
+func logSnapshotIfVerbose(log logr.Logger, s *cache.Snapshot) {
+	if logV := log.V(5); logV.Enabled() {
```

Comment:
IIRC, our max log level is `6`. So is it better to use `V(6)` here and Line 37 as well?

https://github.com/kubernetes-sigs/kueue/blob/2fcdb56f171bfde4f03584fc7ddd7b626e8e8846/pkg/controller/jobframework/reconciler.go#L335